kate/src/lib.rs

```diff
@@ -3,7 +3,7 @@
 
 #[cfg(feature = "std")]
 pub mod com;
-#[cfg(feature = "std")]
+#[cfg(any(feature = "std", feature = "serde"))]
```

Comment:
This doesn't look correct. Why would "serde" enable gridgen?

Reply:
the verify multiproof will be moved to avail core anyway, so this is doable.